src/components/Permissions/AppList/AppList.jsx

```diff
@@ -17,43 +17,34 @@ import ListItemSecondaryAction from 'cozy-ui/transpiled/react/MuiCozyTheme/ListI
 import NavigationList, {
   NavigationListSection
 } from 'cozy-ui/transpiled/react/NavigationList'
-import { isQueryLoading, hasQueryBeenLoaded } from 'cozy-client'
-import { buildAppsQuery, buildKonnectorsQuery } from 'lib/queries'
-import { useQuery } from 'cozy-client'
+import useAppsOrKonnectors from '../hooks/useAppsOrKonnectors'
 import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
 
 const AppList = () => {
   const { t } = useI18n()
-  const appsQuery = buildAppsQuery()
-  const queryResultApps = useQuery(appsQuery.definition, appsQuery.options)
-  const konnectorsQuery = buildKonnectorsQuery()
-  const queryResultKonnectors = useQuery(
-    konnectorsQuery.definition,
-    konnectorsQuery.options
-  )
   const { isMobile, isTablet } = useBreakpoints()
+
+  const { isResulsLoading, hasQueryFailed, appsResult, konnectorsResult } =
```

Comment:
isResul**t**Loading

Reply:
@zatteo I didn't name `isResultLoading` because there is two result this time `appsResult` and `konnectorsResult` but this is maybe confusing

Reply:
Oh maybe there is a better name. Maybe areResultsLoading ? But it is already okay for me now. isResulsLoading is confusing for me.